### PR TITLE
Version 1.20.1 → 1.21.0

### PR DESCRIPTION
**Pull request by Gabriel439**

### CODE REVIEW

File: dhall/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,105 @@
...
+    * This simplifies integration with other command-line tools (such as file
+      watchers)
+    * See: https://github.com/dhall-lang/dhall-haskell/pull/795
+    * See: https://github.com/dhall-lang/dhall-haskell/pull/803
+* Feature: `dhall freeze` now only freezes remote imports by default
+    * `dhall freeze` used to freeze all imports (including local imports and
+      environment variables)
+    * Now `dhall freeze` only freezes remote imports by default, which is what
+      most users want
+    * You can install freeze all imports using the `--everything` flag
```
Comment (jneira) [inline dhall/CHANGELOG.md]: I think we finally used `--all` instead `--everything`

File: dhall/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,105 @@
...
+* Supports version 6.0.0 of the language standard
+    * See: https://github.com/dhall-lang/dhall-lang/releases/tag/v6.0.0
+* BREAKING CHANGE: Remove the `constructors` keyword
+    * ... as standardized in version 6.0.0 of the language standard
+    * The deprecation cycle is over, so the keyword is no longer supported
+    * For more details, see: https://github.com/dhall-lang/dhall-lang/wiki/Migration%3A-Deprecation-of-constructors-keyword
+    * See: https://github.com/dhall-lang/dhall-haskell/pull/829
+* BREAKING CHANGE: CBOR-encode only special `Double`s as half-floats
+    * ... as standardized in version 6.0.0 of the language standard
+    * CBOR `Double`s except `Infinity`/`-Infinity`/`NaN` are now encoded in at
```
Comment (f-f) [inline dhall/CHANGELOG.md]: We should add `0.0` to this list of values, as it's encoded as half-float too

**Comment by Gabriel439** [comment]
@f-f: Yes.  I'm including that change in this release

**Comment by f-f** [comment]
Thank you! 🙂

**Comment by Gabriel439** [comment]
@f-f: You're welcome!

**Pull request merged**